literal/decimal_util.go

```diff
@@ -0,0 +1,68 @@
+package literal
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/cockroachdb/apd"
```

Comment:
If we're going to use this, we should use the most recent version (https://pkg.go.dev/github.com/cockroachdb/apd/v3) please change the import accordingly (add the `/v3`)

Reply:
done